unstructured_platform_plugins/schema/json_schema.py

```diff
@@ -306,7 +312,9 @@ def schema_to_base_model(schema: dict, name: str = "reconstructed_model") -> Typ
                     )
                     for index, type_info in enumerate(any_of_entries)
                 ]
-                t = Union[*type_info]
+                # To support python3.10, unpacking not supported
+                # t = Union[*type_info]
+                t = _UnionGenericAlias(Union, tuple(type_info))
```

Comment:
I don't understand comment in line 316 - is that logical abstraction of line 317?

Reply:
Line 316 is the code that would work if we dropped support for `3.10` which doesn't support unpacking the list inside the square brackets.

Reply:
Ok, then to make it 100% clear I propose sth like
```
# To support python3.10 we need to use _UnionGenericAlias, because unpacking the list is not supported
# Otherwise we could do it like: t = Union[*type_info]
t = _UnionGenericAlias(Union, tuple(type_info))
```